src/main/java/uk/ac/cam/cl/dtg/segue/scheduler/jobs/DeleteEventAdditionalBookingInformationJob.java

```diff
@@ -0,0 +1,120 @@
+        try {
+            ResultsWrapper<ContentDTO> findByFieldNames = this.contentManager.findByFieldNames(
+                    properties.getProperty(CONTENT_INDEX), ContentService.generateDefaultFieldToMatch(fieldsToMatch),
+                    startIndex, limit, sortInstructions, filterInstructions);
+            for (ContentDTO contentResult : findByFieldNames.getResults()) {
+                if (contentResult instanceof IsaacEventPageDTO) {
+                    IsaacEventPageDTO page = (IsaacEventPageDTO) contentResult;
+                    // Event end date (if present) > 30 days ago, else event date > 30 days ago
+                    if ((page.getEndDate() != null && page.getEndDate().toInstant().isBefore(thirtyDaysAgo.toInstant())) || (page.getDate().toInstant().isBefore(thirtyDaysAgo.toInstant()))) {
+                        try (Connection conn = database.getDatabaseConnection()) {
+                            PreparedStatement pst;
+                            // Check for additional info that needs removing, check if pii has already been removed, if
+                            // so, don't re-remove
+                            pst = conn.prepareStatement(
+                                    "UPDATE event_bookings SET additional_booking_information=jsonb_set(jsonb_set(jsonb_set(jsonb_set(additional_booking_information," +
+                                    " '{emergencyName}', '\"[REMOVED]\"'::JSONB, FALSE)," +
+                                    " '{emergencyNumber}', '\"[REMOVED]\"'::JSONB, FALSE)," +
+                                    " '{accessibilityRequirements}', '\"[REMOVED]\"'::JSONB, FALSE)," +
+                                    " '{medicalRequirements}', '\"[REMOVED]\"'::JSONB, FALSE)," +
+                                    " pii_removed=? " +
+                                    " WHERE event_id = ?" +
+                                    " AND additional_booking_information ??| array['emergencyName', 'emergencyNumber', 'accessibilityRequirements', 'medicalRequirements']" +
+                                    " AND pii_removed IS NULL");
+                            pst.setTimestamp(1, new Timestamp(new Date().getTime()));
+                            pst.setString(2, page.getId());
+
+                            int affectedRows = pst.executeUpdate();
+                            if (affectedRows > 0) {
+                                log.info("Event " + page.getId() + " had " + affectedRows + " bookings which have been scrubbed of PII");
+                            }
+                        }
+                    }
+                }
+            }
+            log.info("Ran DeleteEventAdditionalBookingInformationJob");
+        } catch (SQLException e) {
+            try {
+                throw new SegueDatabaseException("Postgres exception", e);
+            } catch (SegueDatabaseException ex) {
+                ex.printStackTrace();
+            }
```

Comment:
If you want to wrap it as a SegueDatabaseException I would have thought that `new SegueDatabaseEception("Postgres exception", e).printStackTrace();` is equivalent to this code